src/Search/AbstractRegexGambit.php

```diff
@@ -13,15 +13,15 @@ abstract class AbstractRegexGambit implements GambitInterface
 {
     /**
      * The regex pattern to match the bit against.
-     *
-     * @var string
      */
-    protected $pattern;
+    protected function getGambitPattern()
```

Comment:
Should we make this abstract and just require it? Probably not in this release for BC reasons, but in the future?

Reply:
absolutely yes